cloud-agent/service/server/src/main/scala/org/hyperledger/identus/pollux/credentialschema/http/CredentialSchemaDidUrlResponse.scala

```diff
@@ -0,0 +1,77 @@
+package org.hyperledger.identus.pollux.credentialschema.http
+
+import org.hyperledger.identus.api.http.*
+import org.hyperledger.identus.castor.core.model.did.{DIDUrl, PrismDID}
+import org.hyperledger.identus.pollux.core.model
+import org.hyperledger.identus.pollux.core.model.schema.CredentialSchema
+import org.hyperledger.identus.pollux.credentialschema.http.CredentialSchemaDidUrlResponse.annotations
+import org.hyperledger.identus.shared.crypto.Sha256Hash
+import org.hyperledger.identus.shared.utils.{Base64Utils, Json as JsonUtils}
+import sttp.model.Uri
+import sttp.model.Uri.*
+import sttp.tapir.Schema
+import sttp.tapir.Schema.annotations.{default, description, encodedExample, encodedName}
+import zio.json.*
+
+import scala.collection.immutable.ListMap
+
+case class CredentialSchemaDidUrlResponse(
+    @description(annotations.resource.description)
+    @encodedExample(annotations.resource.example)
+    resource: String,
+    @description(annotations.schemaUrl.description)
+    @encodedExample(annotations.schemaUrl.example)
+    schemaUrl: String,
+)
+
+object CredentialSchemaDidUrlResponse {
+
+  def fromDomain(cs: CredentialSchema, serviceName: String): Either[String, CredentialSchemaDidUrlResponse] = {
+
+//    Url : TODO: change in codebase with DIDUrl usage
+//      .parse(s"${jwtIssuer.did}?resourceService=$statusListRegistryServiceName&resourcePath=$resourcePath#$segment")
+//      .toString
+//    ,
+    for {
+      authorDid <- PrismDID.fromString(cs.author)
```

Comment:
@shotexa, please ignore my previous comment about merging the response types.
As for me, we should isolate two endpoints from each other as these are two different specs we offer.